tests/ImageSharp.Interactive.Tests/FormatterTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.Threading.Tasks;
+using AngleSharp.Html.Parser;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Microsoft.DotNet.Interactive.Formatting;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Drawing.Processing;
+using SixLabors.ImageSharp.Interactive;
+using SixLabors.ImageSharp.PixelFormats;
+using SixLabors.ImageSharp.Processing;
+using Xunit;
+
+namespace ImageSharp.Interactive.Tests
+{
+    public class FormatterTests : IDisposable
+    {
+        public FormatterTests() => KernelExtension.RegisterFormatters();
+
+        [Fact]
+        public async Task Formats_image_as_png()
```

Comment:
Lets use standard casing for test names. My OCD Gets triggered otherwise 😉

Reply:
ok